### PR TITLE
Create a KB for incorrect replica expansion

### PR DESCRIPTION
**Pull request by ejweber**
longhorn/longhorn#6391

### CODE REVIEW

**Review by innobead — DISMISSED** [review]
In general, LGTM. Very detailed KB, well writen. 

After Phan or James review, we can merge this.

File: content/kb/troubleshooting-unexpected-expansion-leads-to-degredation-or-attach-failure.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+Complete fix planned in:
+
+- Longhorn v1.4.x
+- Longhorn v1.5.x
+- Longhorn v1.6.0
+
+## Symptoms
+
+While the root cause is always the same, symptoms can vary depending on other factors (e.g. whether there are multiple
```
Comment (innobead) [inline content/kb/troubleshooting-unexpected-expansion-leads-to-degredation-or-attach-failure.md:33]:
```suggestion
While the root cause is always the same, symptoms can vary depend on other factors (e.g. whether there are multiple
```

File: content/kb/troubleshooting-unexpected-expansion-leads-to-degredation-or-attach-failure.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+If symptoms are observed and there is an acceptable backup,
+[restore from backup](../../docs/1.5.1/snapshots-and-backups/backup-and-restore/restore-from-a-backup).
+
+If symptoms are observed and there is not an acceptable backup, expand the volume to the size of the expanded replica.
+
+1. Identify the size of the expanded replica. This is the larger size shown in the UI error and the instance-manager
+   logs. It is also the larger size of the snapshot files on disk.
+1. Scale down the workload. It is likely already not running due to the issue.
+1. [Expand the volume to the larger size](../../docs/1.5.1/volumes-and-nodes/expansion).
```
[the remark this answers is not in the document]
Comment (ejweber) [reply content/kb/troubleshooting-unexpected-expansion-leads-to-degredation-or-attach-failure.md]: You are right. My first dealings with the issue (when I initially wrote the workaround steps) were in v1.3.2. I tried this on master-head both through the UI and with a PVC and it worked.

I think there are some manifestations of the issue where this wouldn't work, so I left in a suggestion to scale down if the user hits an issue.

**Review by PhanLe1010 — APPROVED** [review]
LGTM 🚀

**Pull request merged**